website/blog/2021-11-05-release.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: release-0.19
+title: "New in Wave 0.19: Custom Colors"
+author: Martin Turoci
+author_title: Software Engineer @ H2O.ai
+author_url: https://github.com/mturoci
+author_image_url: https://avatars.githubusercontent.com/u/64769322?v=4
+tags: [release]
+---
+
+With the latest release, we are excited to announce Wave becoming more colorful! In addition to existing color themes (`default`, `neon`) we added `h2o-dark` and plan on expanding the list to at least a dozen
+so stay tuned. If interested, you can track the progress [here](https://github.com/h2oai/wave/issues/696). However, having only predefined color palettes is limiting so we decided to allow a custom, user-defined palettes.
+
+<!--truncate-->
+
+## How it works?
+
+Firstly, a new `ui.theme` needs to be registered within the `themes` attribute in `ui.meta_card`. It takes the name of the theme and 4 main colors as arguments.
```

Comment:
Consider explaining explicitly what the 4 main colors are, what's the right one to use for a fun accent, which should be more solid colors?

Reply:
Added a description for each color, but not sure what is meant by the "fun accent". The general theme impression is based on colors altogether or am I missing something?